### PR TITLE
Make flaky network test less flaky

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: integration/tests/api/docker_test.go [hunk 1]
```diff
@@ -253,6 +253,9 @@ func TestDockerContainerNetworkStats(t *testing.T) {
 	containerId := fm.Docker().RunBusybox("watch", "-n1", "wget", "https://www.google.com/")
 	waitForContainer(containerId, fm)
 
+	// Add a little sleep to let the request complete a few times
```
Comment (vishh) [inline integration/tests/api/docker_test.go]: What request? `docker run`? `waitForContainer` is supposed to ensure that the container is running right?
Are you trying to give some time for cadvisor to gather metrics?

Comment (jimmidyson) [inline integration/tests/api/docker_test.go]: The `wget` req on line 253 (https://github.com/google/cadvisor/pull/993/files#diff-6333e565f1256ee66767acf24186c52cR253). That's what should be causing network traffic. The problem I think that's happening is there's a race between container is up check via `waitForContainer` & first `wget` requests.

**Comment by jimmidyson** [comment]
Closing - this doesn't seem to have helped at all. Wondering now if the test is actually valid?

**Comment by vishh** [comment]
That test needs some more love and care :(

**Pull request closed without merging**